cmd/krel/cmd/release_notes.go

```diff
@@ -115,60 +126,38 @@ func init() {
 	)
 
 	releaseNotesCmd.PersistentFlags().StringVar(
-		&releaseNotesOpts.draftOrg,
-		"draft-org",
+		&releaseNotesOpts.GithubOrg,
+		"org",
 		"",
-		"a Github organization owner of the fork of k/sig-release where the Release Notes Draft PR will be created",
-	)
-
-	releaseNotesCmd.PersistentFlags().StringVar(
-		&releaseNotesOpts.draftRepo,
-		"draft-repo",
-		git.DefaultGithubReleaseRepo,
-		"the name of the fork of k/sig-release, the Release Notes Draft PR will be created from this repository",
+		"a Github organization hosting the forks of k/sig-release or ksigs/release-notes",
 	)
 
 	releaseNotesCmd.PersistentFlags().BoolVar(
 		&releaseNotesOpts.createDraftPR,
 		"create-draft-pr",
 		false,
-		"create the Release Notes Draft PR. --draft-org and --draft-repo must be set along with this option",
-	)
-
-	releaseNotesCmd.PersistentFlags().StringVar(
-		&releaseNotesOpts.websiteOrg,
-		"website-org",
-		"",
-		"a Github organization owner of the fork of kuberntets-sigs/release-notes where the Website PR will be created",
-	)
-
-	releaseNotesCmd.PersistentFlags().StringVar(
-		&releaseNotesOpts.websiteRepo,
-		"website-repo",
-		"release-notes",
-		"the name of the fork of kuberntets-sigs/release-notes, the Release Notes Draft PR will be created from this repository",
+		"Update the Release Notes draft and create a PR in k/sig-release",
 	)
 
 	releaseNotesCmd.PersistentFlags().BoolVar(
 		&releaseNotesOpts.createWebsitePR,
 		"create-website-pr",
 		false,
-		"generate the Releas Notes to a local fork of relnotes.k8s.io and create a PR.  --draft-org and --draft-repo must be set along with this option",
+		"Patch the relnotes.k8s.io sources and generate a PR with the changes",
```

Comment:
I think we should start each help with lower or upper case (I think lower case is our default).